### PR TITLE
Switch components from headlessui or radix to blueprint

### PR DESCRIPTION
**Pull request by wadjih-bencheikh18**
closes : https://github.com/zakodium-oss/react-science/issues/469

### CODE REVIEW

File: src/components/color-picker/gradient-select/GradientSelect.tsx [hunk 1]
```diff
@@ -53,86 +54,60 @@ export interface GradientSelectProps {
   value: GradientScaleName;
   onChange: (value: GradientScaleName) => void;
 }
-
+const renderFilm: ItemRenderer<GradientScaleName> = (
```
Comment (targos) [inline src/components/color-picker/gradient-select/GradientSelect.tsx:57]: why "Film" ?

Comment (wadjih-bencheikh18) [reply src/components/color-picker/gradient-select/GradientSelect.tsx:57]: Oh, sorry I forgot to rename it.
It's fromthe documentation example

File: src/components/color-picker/gradient-select/GradientSelect.tsx [hunk 2]
```diff
@@ -1,7 +1,8 @@
 /** @jsxImportSource @emotion/react */
 import { css } from '@emotion/react';
 import styled from '@emotion/styled';
-import * as Select from '@radix-ui/react-select';
+import { Button, MenuItem } from '@blueprintjs/core';
+import { ItemRenderer, Select as BlueprintSelect } from '@blueprintjs/select';
```
Comment (stropitek) [inline src/components/color-picker/gradient-select/GradientSelect.tsx:5]: Don't rename the import if there is no name conflict.

File: src/components/color-picker/gradient-select/GradientSelect.tsx [hunk 3]
```diff
@@ -53,86 +54,60 @@ export interface GradientSelectProps {
...
+) => {
+  if (!modifiers.matchesPredicate) {
+    return null;
+  }
+  return (
+    <MenuItem
+      active={modifiers.active}
+      disabled={modifiers.disabled}
+      key={option}
+      // label={option}
```
Comment (stropitek) [inline src/components/color-picker/gradient-select/GradientSelect.tsx:69]: to be removed

**Review by stropitek — APPROVED** [review]
LGTM module some nits

**Pull request merged**